storage-service/src/main/java/com/stnetix/ariaddna/persistence/utils/JPAConfiguration.java

```diff
@@ -29,7 +29,7 @@
     public DataSource dataSource(){
         XmlDbSettingHandler handler = null;
         try {
-            handler = (XmlDbSettingHandler) new XmlParser("settings.xml", new XmlDbSettingHandler()).getHandler();
+            handler = (XmlDbSettingHandler) new XmlParser("./../settings.xml", new XmlDbSettingHandler()).getHandler();
```

Comment:
It is not good idea have "hardcode path". I think that application should be searching configuration file in classpath or as variant #2 soome predefined path set (work directory, /etc/<appname>/)